src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -6480,6 +6474,21 @@ private boolean isNewDoc()
       return docUpdateSentinel_.getPath() == null;
    }
    
+   private static boolean shouldEnforceHardTabs(FileSystemItem item)
+   {
+      if (item == null)
+         return false;
+      
+      if ("Makefile".equals(item.getName()) ||
+          "Makevars".equals(item.getName()) ||
+          ".tsv".equals(item.getExtension()))
```

Comment:
Do we need to limit this to `.in`/`.win` for `Makefile` as we did before?

Reply:
Yes; thanks for catching that!